### PR TITLE
Added logging for getdevicedata

### PR DESCRIPTION
**Pull request by ehaligow**
Fix for issue #20

### CODE REVIEW

**Comment by ehaligow** [comment]
Fix for issue #20

File: src/grpcAPIs.go [hunk 1]
```diff
@@ -651,6 +651,21 @@ func (s *Server) GetDeviceData(c context.Context, device *manager.Device) (*mana
 	if device == nil || len(device.IpAddress) == 0 {
 		return nil, status.Errorf(http.StatusBadRequest, ErrDeviceData.String())
 	}
+	if !s.devicemap[device.IpAddress].QueryState {
+		logrus.Errorf(ErrCollectingNotStarted.String())
+		return nil, errors.New(ErrCollectingNotStarted.String())
+	}
+
+	found := false
+	for _, api := range s.devicemap[device.IpAddress].RfAPIList {
```
Comment (MaciejMis) [inline src/grpcAPIs.go:660]: This check whether ip address is on a list could be extracted to a method.

Comment (ehaligow) [reply src/grpcAPIs.go]: Done.

**Comment by MaciejMis** [comment]
This PR was approved at the OPENRMC-DM meeting.

**Pull request merged**